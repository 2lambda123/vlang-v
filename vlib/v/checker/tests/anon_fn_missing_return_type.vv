fn main() {
	f := fn (i int) int { println(i) }
	f(0)
}
