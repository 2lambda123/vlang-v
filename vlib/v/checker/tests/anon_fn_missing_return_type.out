vlib/v/checker/tests/anon_fn_missing_return_type.vv:2:7: error: missing return at end of function `anon_49_7_7`
    1 | fn main() {
    2 |     f := fn (i int) int { println(i) }
      |          ~~
    3 |     f(0)
    4 | }
